featDetectorWrapper/detectedObj[] detected_classes
